docs/feature-compatibility.txt

```diff
@@ -0,0 +1,331 @@
+.. _laravel-feature-compat:
+
+=============================
+Laravel Feature Compatibility
+=============================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: php framework, odm, support
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+This guide describes the Laravel features that are supported by
+the {+odm-long+}. This page discusses Laravel version 11.x feature
+availability in the {+odm-short+}.
+
+The following sections contain tables that describe whether individual
+features are available in {+odm-short+}.
+
+Database Features
+-----------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Eloquent Feature
+     - Availability
+
+   * - Configuration
+     - ✓
+
+   * - Read/Write Connections
+     - Use :manual:`read preference </core/read-preference/>` instead.
+
+   * - Multiple Database Connections
+     - ✓
+
+   * - Listening for Query Events
+     - ✓
+
+   * - Monitoring Cumulative Query Time
+     - ✓
+
+   * - Transactions
+     - ✓ See :ref:`laravel-transactions`.
+
+   * - Command Line Interface (CLI)
+     - Use the :mdb-shell:`MongoDB Shell <>` (``mongosh``).
+
+   * - Database Inspection
+     - *Unsupported*
+
+   * - Database Monitoring
+     - *Unsupported*
+
+.. TODO Listening to query events: yes (to be checked)
+.. TODO Monitoring cumulative query time: yes (to be checked)
```

Comment:
I: The [Laravel / PHP workflow doc](https://docs.google.com/document/d/12mM87pjVzcTkPCbei2c6Ollb-xNxPQYATHVuavpsRxc/edit#bookmark=id.6ijzkbvfyx5x) recommends avoiding these rST placeholders, so I'd remove them

Reply:
Also applies to the other TODOs on this page

Reply:
i will just remove all of the to be checked lines!